.travis.yml

```diff
@@ -55,6 +57,7 @@ before_install:
   - if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ] && [ "${TRAVIS_PHP_VERSION:0:3}" != "5.3" ]; then composer require --no-update --dev league/flysystem:~1.0; fi;
   - if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ] && [ "${TRAVIS_PHP_VERSION:0:1}" != "7" ]; then yes "" | pecl -q install -f mongo; composer require --no-update --dev doctrine/mongodb-odm:~1.0; fi;
   - if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ] && [ "${TRAVIS_PHP_VERSION:0:1}" == "7" ]; then yes "" | pecl -q install -f mongodb; travis_retry composer require --dev alcaeus/mongo-php-adapter:~1.0; composer require --no-update --dev doctrine/mongodb-odm:~1.0; fi
+  - if [ "$WITH_ENQUEUE" = true ] ; then composer require --no-update --dev enqueue/enqueue-bundle:^0.3.6; fi;
```

Comment:
Why not enable this test globally for all supported PHP versions? Or at least all PHP 7.x runs? I think this would be a better approach to ensure no incompatibilities are introduced across Symfony versions. We already have some Symfony 2.32, `2.8`, `3.2` and `3.3` specific features or toggled usages.

**All PHP `5.6` and PHP `7.x` Runs**

```bash
if [ "${TRAVIS_PHP_VERSION:0:3}" == "5.6" ] || [ "${TRAVIS_PHP_VERSION:0:1}" == "7" ]; then [...] fi
```
**All PHP `7.x` Runs**

```bash
if [ "${TRAVIS_PHP_VERSION:0:1}" == "7" ]; then [...] fi
```